packages/dd-trace/test/appsec/iast/utils.js

```diff
@@ -106,4 +112,12 @@ function testThatRequestHasNotVulnerability (app, vulnerability) {
   testInRequest(app, tests)
 }
 
-module.exports = { testThatRequestHasNotVulnerability, testThatRequestHasVulnerability, testInRequest }
+function copyFileToTmp (src) {
+  const prefix = Math.floor(Math.random() * 1000000)
```

Comment:
If you create a random prefix out of the method, a here, only increase a counter, you'll have a prefix imposible to duplicate in the same execution.
I know that is nearly imposible, but you can have same Math.random() result, but you can't have same id if you increment a counter.